modules/katamari/src/test/ts/atomic/api/struct/AdtTest.ts

```diff
@@ -25,7 +25,7 @@ UnitTest.test('ADT Test', () => {
   };
 
   const checkInvalidGenerate = (cases, message) => {
-    checkInvalid('generate() did not throw an error. Input: "' + message + '\'', () => {
+    checkInvalid('generate() did not throw an error. Input: "' + message + '"', () => {
```

Comment:
Note that the change from ' to " here on the closing quote was deliberate as I considered it a typo.